lib/ace/mode/sqlserver_highlight_rules.js

```diff
@@ -164,14 +154,11 @@ var SqlServerHighlightRules = function() {
             start: "/\\*",
             end: "\\*/"
         }, {
-            token: "string", // ' string
```

Comment:
I think this was the unnecessary rule you were referring to, correct? I think I copied that rule from somewhere else